frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -39,7 +39,7 @@ export const DashboardLoadingAndErrorWrapper = styled(
 export const DashboardStyled = styled.div`
   display: flex;
   flex-direction: column;
-  height: 100%;
+  min-height: 100%;
```

Comment:
This, combined with the above, means we're always at minimum using the full screen, while still allowing scrolling to happen if dashboards are larger